src/release/breaking-changes/flutter-memory-allocations.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Deprecate MemoryAllocations in favor of FlutterMemoryAllocations
+description: >-
+  MemoryAllocations is renamed to FlutterMemoryAllocations
+---
+
+## Summary
+
+Disposables in pure Dart projects cannot use MemoryAllocations in Flutter. 
+So, to be leak trackable they need a Dart-only class.
+MemoryAllocations in Flutter is renamed to release the name
+for Dart-only class.
+
+## Migration guide
+
+Before:
+
+```dart 
+  MemoryAllocations. ...
```

Comment:
I would extend this to give some examples of what people would be calling on MemoryAllocations and how how that's now called on FlutterMemoryAllocations.